src/components/schedule-page/table/schedule/schedule-parts/shift-cell/shift-cell.component.tsx

```diff
@@ -109,7 +113,14 @@ export function ShiftCellComponentF(options: ShiftCellOptions): JSX.Element {
         selection: isSelected || (isComponentVisible && isBlocked),
         blocked: isBlocked,
       })}
-      onClick={(): void => toggleComponentVisibility()}
+      onClick={(): void => {
+        toggleComponentVisibility();
+      }}
+      onMouseLeave={(): void => {
+        setTimeout(() => {
+          setIsComponentVisible(false);
```

Comment:
Proponuję dodać hook zależny od scrolowania. 
I w przypadku jeśli ktoś scroluje, to ukrywać ten dropdown. 

Bo teraz trochę przy scrolowaniu trochę dziwnie to się zachowuje
![image](https://user-images.githubusercontent.com/21079319/111079157-fad5bf00-84f8-11eb-893c-e78f7975e562.png)

@prenc , co ty o tym myślisz?